substrait_consumer/producers.py

```diff
@@ -138,6 +153,86 @@ def name(self):
         return "IsthmusProducer"
 
 
+class DataFusionProducer:
+    """
+    Adapts the DataFusion Substrait producer to the test framework.
+    """
+    def __init__(self, db_connection=None):
+        self._ctx = SessionContext()
+        if db_connection is not None:
+            self._db_connection = db_connection
+        else:
+            self._db_connection = db_connection
+
+    def set_db_connection(self, db_connection):
+        self._db_connection = db_connection
+
+    def produce_substrait(self, sql_query: str, ibis_expr: str = None) -> str:
```

Comment:
ibis_expr is not documented below

Reply:
ibis_expr is only used with the IbisProducer